actix-files/src/lib.rs

```diff
@@ -317,40 +334,31 @@ where
     }
 }
 
-impl<P, C: StaticFileConfig + 'static> NewService for Files<P, C> {
+impl<P> NewService for Files<P> {
     type Request = ServiceRequest<P>;
     type Response = ServiceResponse;
     type Error = Error;
-    type Service = FilesService<P, C>;
+    type Service = Self;
     type InitError = ();
     type Future = FutureResult<Self::Service, Self::InitError>;
 
     fn new_service(&self, _: &()) -> Self::Future {
-        ok(FilesService {
+        ok(Files {
```

Comment:
should `Files` just implement Clone trait?

Reply:
Yeah about that, it turned out that for some reason I cannot properly clone it.

I think it might be because of generic `P` without `Clone` bounds, I'm not sure if I can add it there?

To clairfy for some reason just Files::clone() returns `&Files`

Reply:
You need manually implement Clone trait

Reply:
Done